CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff

```diff
@@ -1266,7 +1266,7 @@ Sie enthält derzeit %2$lld pins.</target>
       </trans-unit>
       <trans-unit id="Pinned items are shown at the top of the main screen when you open Callsheet. You can pin movies, shows, or people." xml:space="preserve">
         <source>Pinned items are shown at the top of the main screen when you open Callsheet. You can pin movies, shows, or people.</source>
-        <target state="translated">Angehäftete Elemente werden oben auf dem Hauptbildschirm angezeigt, wenn Sie Callsheet öffnen. Sie können Filme, Serien oder Personen anheften.</target>
+        <target state="translated">Angeheftete Elemente werden oben auf dem Hauptbildschirm angezeigt, wenn du Callsheet öffnest. Du kannst Filme, Serien oder Personen anheften.</target>
```

Comment:
"Angehäftete": Typo
Sie → du: In German there are two ways to address someone, a more formal one ("Sie") and a more casual one ("du"). The latter is more common in indie apps and you have that form in other strings as well.